packages/client/src/app/services/user/prose.service.ts

```diff
@@ -0,0 +1,77 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable, throwError } from 'rxjs';
+import { map, catchError } from 'rxjs/operators';
+import { FileUploader, FileItem, ParsedResponseHeaders } from 'ng2-file-upload';
+
+import { ProseContent, CreateProse } from '@pulp-fiction/models/content';
+import { HttpError } from '../../models/site';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ProseService {
+  private url = `/api/content/prose`;
+
+  constructor(private http: HttpClient) { }
+
+  public createProse(proseInfo: CreateProse) {
+    return this.http.put<ProseContent>(`${this.url}/create-prose`, proseInfo, {observe: 'response', withCredentials: true})
+      .pipe(map(res => {
+        return res.body;
+      }), catchError(err => {
+        return throwError(err);
+      }));
+  }
+
+  public editProse(contentId: string, proseInfo: CreateProse) {
+    return this.http.patch<ProseContent>(`${this.url}/edit-prose?contentId=${contentId}`, proseInfo, {observe: 'response', withCredentials: true})
+      .pipe(map(res => {
+        return res.body;
+      }), catchError(err => {
+        return throwError(err);
+      }));
+  }
+
+  public uploadCoverart(uploader: FileUploader) {
```

Comment:
this isn't something I'd block on, but this function can probably be shared between prose and poetry